test/utils/helpers.ts

```diff
@@ -119,3 +121,18 @@ export function isAddressInfo(address: unknown): address is net.AddressInfo {
     (address as net.AddressInfo).address !== undefined
   )
 }
+
+/**
+ * If using a proxy, return the address of the proxy.
+ *
+ * Otherwise, return the direct address of code-server.
+ */
+export async function getMaybeProxiedCodeServer(codeServer: CodeServerPage | CodeServer): Promise<string> {
+  let address = await codeServer.address()
```

Comment:
apologies for a last-minute nit, feel free to ignore:

```suggestion
  const address = await codeServer.address()
```

Probably we should add an eslint check for let/const.

Reply:
Good call!